static/src/stylesheets/_vars.scss

```diff
@@ -86,23 +86,28 @@ $garnett-neutral-6: #767676;
 $garnett-neutral-7: #444444;
 
 $news-garnett-main-1: #e00000; //used for kicker
+$news-garnett-main-2: #ffbac8;
```

Comment:
Is this colour inherent to news, or any of the other pillars? It looks more like it’s inherent to the analysis design type, so could probably be defined in an analysis type scss partial?

Meta: I’m about to take a flight, so take these comments as “food for thought”... don’t let my comments hold you up / prevent you from merging!

Reply:
one for @blongden73 maybe?

Reply:
Yep, totally!

Reply:
Maybe we can follow the pattern we have for the `feature` type headline if it's specific to to the `analysis` type underline. eg. `$news-garnett-analysis`,  `opinion-garnett-analysis`, `arts-garnett-analysis` etc...